apps/music/js/ui/tab_bar.js

```diff
@@ -15,25 +14,20 @@ var TabBar = {
     null
   ],
 
-  get view() {
-    return document.getElementById('tabs');
+  get views() {
+    return document.getElementsByClassName('tabs');
```

Comment:
I planned to make minimum changes on master for NGA, but if we will have another NGA branch(like what you said in the sync-up meeting), then probably your suggestion is a better choice and good for long-term, though I'll have to make more changes and we need to think a way on how to communicate with all the `TabBar` instances.